servicetalk-loadbalancer/src/main/java/io/servicetalk/loadbalancer/LoadBalancers.java

```diff
@@ -23,7 +23,7 @@
 /**
  * A factory to create {@link DefaultLoadBalancer DefaultLoadBalancers}.
  */
-final class LoadBalancers {
+public final class LoadBalancers {
```

Comment:
I think we need to do this eventually but I'm not seeing how it's related to this change.